src/Kaixa.groovy

```diff
@@ -1367,12 +1368,17 @@ public class Kaixa {
 	 * @method typeInto
 	 * @param {TestObject|String} item - the TestObject or CSS selector of interest
 	 * @param {String} text - the text to type
+	 * @param {boolean} newline - if newline should be added after text
 	 */
-	public static void typeInto(Object item, Object text) {
+	public static void typeInto(Object item, Object text, boolean newline = false) {
 		Kaixa.log('⌨️ Type "' + text + '" into ' + item);
 		TestObject obj = Kaixa.ensureTestObject(item);
 		Kaixa.waitForElementVisible(obj);
 		WebUI.setText(obj, text.toString());
+		// Workaround for typing "\n" not working in Safari and Firefox.
+		if (newline) {
```

Comment:
Could we change this to `newline || text.toString().endsWith('\n')`?

Also, I'll need to bump the version number of Kaixa.